[rjsf] Styles fixes

harkiratsm opened the pull request:
Signed-off-by: harkiratsm <multaniharry714@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**
Two Columns Layout.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.
 

<!--
Thank you for contributing to Meshery! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

leecalcote commented on ui/themes/rjsf.js:
```diff
@@ -38,13 +38,21 @@ export const rjsfTheme = createTheme({
       root : {
         whiteSpace : "nowrap",
-        // overflow : "hidden",
+        overflow : "hidden",
```
Good update. 👍

leecalcote commented on ui/themes/rjsf.js:
```diff
@@ -38,13 +38,21 @@ export const rjsfTheme = createTheme({
         }
+      },
+      shrink : {
+        maxWidth : "133%",
```
Good update

leecalcote commented on ui/themes/rjsf.js:
```diff
@@ -38,13 +38,21 @@ export const rjsfTheme = createTheme({
+    },
+    MuiFormControl : {
+      root : {
+        marginTop : "0.1em"
```
Hmm. I don't know that this is desirable.

harkiratsm commented:
Newer Changes - 
1) Two Columns Layout for the Design Configurator 
& fix three items from this https://app.clickup.com/t/2wnamcr

leecalcote commented on ui/themes/rjsf.js:
```diff
@@ -152,6 +160,14 @@ export const rjsfTheme = createTheme({
+    },
+    MuiIconButton : {
+      root : {
+        cursor : "default"
```
Good.

leecalcote commented on ui/themes/rjsf.js:
```diff
@@ -32,19 +32,27 @@ export const rjsfTheme = createTheme({
+        overflow : "hidden",
         textOverflow : "ellipsis",
         maxWidth : "75%",
         height : "100%",
```
100% = good.

leecalcote commented on ui/themes/rjsf.js:
```diff
@@ -152,6 +160,14 @@ export const rjsfTheme = createTheme({
         }
       },
+      "grid-xs-6" : {
+        paddingRight : '0.5rem'
```
display: flex; justify-content: space-around;

harkiratsm commented:
This PR fixes - 
1) Fixed the behaviour when input field .
2) On expand the items/object/array fields are not expanding.
3) Be consistent with cursor on tooltip - default pointer (arrow).
4) Apply a bottom margin to name and namespace + no overlapping.
5) Fixed labels and annotations behaviour in Design Configurator .
...... typing

leecalcote commented:
@harkiratsm signal after today's feedback has been incorporated.

harkiratsm commented:
Newer Changes are - scaling the fields using ```space-around``` .

harkiratsm commented:
@leecalcote

leecalcote commented:
@harkiratsm provide a revised, itemized list of changes, please.

harkiratsm commented:
> @harkiratsm provide a revised, itemized list of changes, please.

https://github.com/meshery/meshery/pull/6360#issuecomment-1277436451

leecalcote commented on ui/themes/rjsf.js:
```diff
@@ -9,6 +9,7 @@ export const rjsfTheme = createTheme({
     },
   },
   typography : {
+    fontFamily : "Qanelas Soft, sans-serif",
```
```suggestion
```

leecalcote commented:
@harkiratsm I don't know what this means: `[ Not accepted currently ]`
